.github/workflows/release_notes_check.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Notes Check
+
+on:
+  pull_request:
+    types:
+      # On by default if you specify no types.
+      - "opened"
+      - "reopened"
+      - "synchronize"
+      # For `skip-label` only.
+      - "labeled"
+      - "unlabeled"
+
+jobs:
+  check-release-notes:
+    name: Check release notes are updated
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for a release notes update
+        uses: frequenz-floss/check-for-changed-files@token-support
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          file-pattern: "RELEASE_NOTES.md"
+          prereq-pattern: "src/**"
+          skip-label: "cmd:skip-release-notes"
+          failure-message: "Missing a release notes update. Please add one or apply the ${skip-label} label to the pull request"
```

Comment:
Side note: does this `${skip-label}` really work? Do you have a sample repo where this is running (and failing) to see how it looks like?

Reply:
Yes, I used it and it worked. Example: https://github.com/frequenz-io/frequenz-actor-fcr-balancing/actions/runs/5715881521